### ISSUE TITLE
not import this package in windows11?

### ISSUE THREAD

**Issue by HuJunnj** [nanomsg/nanomsg#1108]
UserWarning: Could not load the default wrapper for your platform: cpy, performance may be affected!
  "%s, performance may be affected!") % (default,))

Who can tell me when the above error message can occur? thanks very much advence!

**Comment by gdamore** [comment]
This doesn't feel like a libnanomsg issue... but possibly a problem with whatever FFI wrapper you're using.

How are you using nanomsg?  (libnanomsg itself -- this project -- is just a C library.)

**Comment by HuJunnj** [comment]
yes, l using nanonsg,But this error message appears when integrating projects with Python，can you help me resolve it, thanks very much advance

BR
Jun



---- Replied Message ----
| From | ***@***.***> |
| Date | 02/04/2024 04:27 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [nanomsg/nanomsg] not import this package in windows11? (Issue #1108) |

This doesn't feel like a libnanomsg issue... but possibly a problem with whatever FFI wrapper you're using.

How are you using nanomsg? (libnanomsg itself -- this project -- is just a C library.)

**Comment by gdamore** [comment]
I have no knowledge about the python wrapper.  Sorry.

**Comment by gdamore** [comment]
I'm closing this.  You should take it up with the maintainer of the python library.
